.docker/docker-compose.yml

```diff
@@ -116,6 +116,51 @@ services:
       - '8284:8284'
     restart: always
 
+  arkadia.notifications:
```

Comment:
are dots valid service names? Did you run the containers with this? Because I thought dots weren't... on the vps I changed them for underscores.

Reply:
Believe it or not, yes, they're valid, and I think I have shared plenty of screenshots in SDS showing that lol